package.json

```diff
@@ -36,6 +36,7 @@
     "node-mysql": "^0.4.2",
     "querystring": "^0.2.0",
     "request": "^2.79.0",
+    "slack-message-builder": "^1.2.1",
```

Comment:
This is fine, but I think we can construct that JSON manually too. Less dependencies is often a better choice.